virttest/virsh.py

```diff
@@ -3329,6 +3329,33 @@ def snapshot_list(name, options=None, **dargs):
     return ret
 
 
+def snapshot_status(name, options=None, **dargs):
```

Comment:
Instead of defining a new function, can you call libvirt_misc.convert_to_dict() to get the information you want?

Reply:
Hi yingshun , Do you mean call the  libvirt_misc.convert_to_dict() after getting response from `def snapshot_list`, but it's weird , the  `def snapshot_list` only return snap name list such as ['s1', 's2', 's3'], this may due to all the snap list content are not oftenly used.

Reply:
Ok, then you can reuse virsh.command to execute snapshot-list subcmd if you dont want to enhance the existing snapshot_list function I think.

Reply:
Ok, then you can reuse virsh.command to execute snapshot-list subcmd if you dont want to enhance the existing snapshot_list function I think.